src/components/Tool/NavToolList/NavToolList.tsx

```diff
@@ -1,61 +1,153 @@
 import React, { Fragment, ReactElement } from 'react';
 import Icon from '@mdi/react';
+import Close from '@mui/icons-material/Close';
 import {
   Box,
+  IconButton,
   List,
   ListItem,
   ListItemIcon,
   ListItemText,
+  Slide,
   Theme,
+  Typography,
 } from '@mui/material';
+import { styled } from '@mui/material/styles';
+import { useTranslation } from 'react-i18next';
 import { makeStyles } from 'tss-react/mui';
+import { useGetToolNotificationsQuery } from 'src/components/Layouts/Primary/TopBar/Items/NavMenu/GetToolNotifcations.generated';
+import { ToolName } from 'src/components/Layouts/Primary/TopBar/Items/NavMenu/NavMenu';
+import { useAccountListId } from 'src/hooks/useAccountListId';
 import { ToolsList } from '../Home/ToolList';
 import { Item } from './Item/Item';
 
 const useStyles = makeStyles()((theme: Theme) => ({
-  list: {
-    width: '290px',
-    transform: 'translateY(55px)',
-    [theme.breakpoints.down('xs')]: {
-      transform: 'translateY(45px)',
-    },
+  root: {
+    overflow: 'hidden',
   },
   li: {
-    borderTop: '1px solid',
-    borderBottom: '1px solid',
-    borderColor: theme.palette.cruGrayDark.main,
+    borderTop: `1px solid ${theme.palette.cruGrayLight.main}`,
+    borderBottom: `1px solid ${theme.palette.cruGrayLight.main}`,
+    borderColor: theme.palette.cruGrayLight.main,
```

Comment:
Is this style necessary? It looks like the border color is set by `borderTop` and `borderBottom`.